.github/workflows/tests.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         uses: shivammathur/setup-php@v2
         with:
           php-version: ${{ matrix.php }}
-          extensions: dom, curl, libxml, mbstring, zip
+          extensions: dom, curl, libxml, mbstring, zip, bcmath
```

Comment:
Would be nice to have these alphabetically sorted.

Reply:
We don't do that anywhere. This is fine.

Reply:
The original code has them sorted?

Reply:
What original code?

Reply:
It's not important. We don't do this anywhere.